stubs/pycparser/c_ast.py

```diff
@@ -344,20 +360,22 @@ def __init__(self, coord: Optional[Coord] = None):
 class Decl(Node):
     name: Optional[str]
     quals: List[str]  # e.g. const
+    align: List[Alignas]
     storage: List[str]  # e.g. register
     funcspec: List[str]  # e.g. inline
-    type: Union_[Type, "Struct", "Union", "Enum"]
+    type: Type
     init: Optional[Union_[Expression, "InitList"]]
     bitsize: Optional[Expression]
 
     def __init__(
         self,
         name: Optional[str],
         quals: List[str],
+        align: List[Alignas],
         storage: List[str],
         funcspec: List[str],
-        type: Union_[Type, "Struct", "Union", "Enum"],
-        init: Optional[Union_[Expression, "InitList"]],
```

Comment:
revert these changes (these fixes to Decl should really be mirrored to m2c, I must have forgot)

Reply:
just these two lines, or all changes to the Decl class?

Reply:
align is still needed, but revert the type and init changes, here and the `type` thing higher up